src/Tasks/Microsoft.NET.Build.Tasks/RunReadyToRunCompiler.cs

```diff
@@ -216,7 +216,11 @@ private string GenerateCrossgen2ResponseFile()
                 result.AppendLine($"--out:\"{_outputR2RImage}\"");
                 if (!String.IsNullOrEmpty(Crossgen2ExtraCommandLineArgs))
                 {
-                    result.AppendLine(Crossgen2ExtraCommandLineArgs);
+                    foreach (string extraArg in Crossgen2ExtraCommandLineArgs.Split(';'))
```

Comment:
Is it possible to use the space as a separator? Note that at present the command line parser does not allow `--option value` format in response files: you have to separate the value with either `:` or `=`.

Reply:
No, we can't use space. It is reasonable for arguments to contain spaces. For instance, one of the cases I specifically want to work with involves passing file names through this mechanism, which on Windows means we need to support spaces.

Reply:
I see. The semicolon is also valid in file names, though definitely less frequently used. Note that at present SDK allows separating extra arguments with `%0a`, e.g., `-p:PublishReadyToRunCrossgen2ExtraArgs="--inputbubble%0a--composite"`.